pkg/resolver/search.go

```diff
@@ -133,6 +142,26 @@ func (s *SearchResult) resolveCount() (int, error) {
 	return count, err
 }
 
+func (s *SearchResult) resolveUids() ([]*string, error) {
+	rows, err := s.pool.Query(context.Background(), s.query, s.params...)
+
+	if err != nil {
+		klog.Errorf("Error resolving query [%s] with args [%+v]. Error: [%+v]", s.query, s.params, err)
+	}
+
+	defer rows.Close()
+	for rows.Next() {
+		var uid string
+		err = rows.Scan(&uid)
+		if err != nil {
+			klog.Errorf("Error %s retrieving rows for query:%s", err.Error(), s.query)
+		}
+		s.uids = append(s.uids, &uid)
+	}
+
+	return s.uids, err
```

Comment:
These returned uids are not being used in line 86. The error is already logged here in line 157.

> _, e := s.resolveUids()

We don't need to return those, in that case.

Reply:
good catch!